app/services/hyrax/qa_select_service_decorator.rb

```diff
@@ -0,0 +1,24 @@
+# OVERRIDE: Hyrax 3.4.2 fixes issues with empty authority and no implicit conversion of string into array
+
+# frozen_string_literal: true
+
+module Hyrax
+  module QaSelectServiceDecorator
+    
+    def active?(id)
+      unless authority.find(id).empty?
+        authority.find(id)&.fetch('active')
```

Comment:
When I'm reading `fetch` I assume that there will be a key error if there's no `active` key in the result.  Should we have a `fetch('active') { false }`?